src/components/LessonCard.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { hot } from 'react-hot-loader';
+import {
+  Card,
+  CardHeader,
+  CardActionArea,
+  SvgIcon,
+} from '@material-ui/core';
+import { CheckCircle, Help, Lens } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+import { grey } from '@material-ui/core/colors';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+  inProgress: {
+    color: theme.palette.primary.main,
+  },
+  complete: {
+    color: theme.palette.success.main,
+  },
+  unavailable: {
+    color: grey[500],
+  },
+}));
+
+const LessonCard = ({ lesson, onClick }) => {
+  const classes = useStyles();
+
+  const { progress } = lesson.state || {};
+  const unavailable = progress === 'UNAVAILABLE';
+
+  const progressIcon = () => {
+    switch (progress) {
+      case 'AVAILABLE':
```

Comment:
This is really slick.
We don't do anything to set the lesson state yet, right?

Reply:
I don't believe so. Remix can set `IN_PROGRESS`, but I don't know if any of the other states can be programatic yet

Reply:
That's exactly what I was thinking -- we can use remix to set it IN_PROGRESS, then hold off worrying about the other ones.

Reply:
I don't believe the API has support to create or set a state. So this will take some more work to do that seems to be outside the scope of the courses view